sites/website/src/pages/index.jsx

```diff
@@ -106,6 +106,20 @@ HelloWorld.define({
                     <CodeBlock language="html">
                         {`<hello-world name="Earth"></hello-world>`}
                     </CodeBlock>
+
+                    <h3>Update your tsconfig.json file to support decorators</h3>
+                    <CodeBlock language="json">
+                      {{
+                        "compilerOptions": {
+                        "experimentalDecorators": true,
+                        "emitDecoratorMetadata": true,
+                        "module": "ESNext",
+                        "moduleResolution": "bundler",
+                        "target": "ES2020",
+                        "useDefineForClassFields": false
+                        }
+                      }}
+                    </CodeBlock>
```

Comment:
Thanks for this! I do wonder if it makes sense to put the tsconfig implementation below `install` in scenarios where folks don't see it working right away without this.

Reply:
Sounds good - to be clear, I think this can go above `npm install` just since it's "required configuration" right now.